Makefile

```diff
@@ -0,0 +1,85 @@
+CONTAINER_NAME=splunk
+CONFIGURATION_BACKUP=.splunk_ta_dataset_config
+
+.PHONY: docker-run
+docker-run:
+	echo "$$(pwd)/output/TA_dataset:/opt/splunk/etc/apps/TA_dataset"
+	docker run -it \
+		-v "$$(pwd)/output/TA_dataset:/opt/splunk/etc/apps/TA_dataset" \
+		-e SPLUNK_START_ARGS=--accept-license \
+		-e SPLUNK_PASSWORD=Test0101 \
+		--platform=linux/amd64 \
+		--name $(CONTAINER_NAME) \
+		-p 8000:8000 \
+		splunk/splunk:latest start
+
+.PHONY: docker-start
+docker-start: docker-run
+
+.PHONY: docker-restart
+docker-restart:
+	docker exec $(CONTAINER_NAME) \
+		sudo -u splunk \
+		/opt/splunk/bin/splunk restart
+
+.PHONY: docker-stop
+docker-stop:
+	docker exec $(CONTAINER_NAME) \
+		sudo -u splunk \
+		/opt/splunk/bin/splunk stop
+
+.PHONY: docker-show-app
+docker-show-app:
+	docker exec $(CONTAINER_NAME) \
+		sudo -u splunk \
+		ls -l /opt/splunk/etc/apps/
+	docker exec $(CONTAINER_NAME) \
+		sudo -u splunk \
+		ls -l /opt/splunk/etc/apps/TA_dataset/
+
+.PHONY: docker-tail-logs
+docker-tail-logs:
+	docker exec $(CONTAINER_NAME) \
+		sudo -u splunk \
+		tail -f \
+			/opt/splunk/var/log/splunk/splunkd.log \
+			/opt/splunk/var/log/splunk/splunkd_stderr.log
+
+.PHONY: docker-bash
+docker-bash:
+	docker exec -i $(CONTAINER_NAME) \
+		sudo -u splunk bash
+
+.PHONY: docker-remove
+docker-remove:
+	docker container rm $(CONTAINER_NAME)
+
+.PHONY: inspect
+inspect:
+	splunk-appinspect inspect TA_dataset --included-tags splunk_appinspect
+
+.PHONY: pack
+pack:
+	mkdir -p $(CONFIGURATION_BACKUP) && \
+	echo "Validate package" && \
+	slim validate TA_dataset && \
+	echo "Preserve configuration" && \
+	if [ -f ./output/TA_dataset/local/ ]; then \
+  		mv -v ./output/TA_dataset/local/* $(CONFIGURATION_BACKUP); \
+  	fi && \
+	rm -rf output/TA_dataset && \
+	echo "Generate package" && \
+	ucc-gen --source TA_dataset && \
```

Comment:
Lets retrieve the version from source code and pass it to ucc-gen command

Reply:
This is the part from the packing process which I do not understand. I have seen at least 2 different places, where some version is specified - but then it's not respected and it looks that you have to specify it as parameter.

Reply:
Fixed